mybot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Effort Controllers ---------------------------------------
  leftRearWheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: left_rear_wheel_hinge
    pid: {p: 100.0, i: 0.1, d: 10.0}

  rightRearWheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: right_rear_wheel_hinge
    pid: {p: 100.0, i: 0.1, d: 10.0}

  leftFrontWheel_effort_controller:
    type: effort_controllers/JointPositionController
    joint: left_steer_joint
    pid: {p: 100.0, i: 0.1, d: 10.0}

  rightFrontWheel_effort_controller:
    type: effort_controllers/JointPositionController
    joint: right_steer_joint
    pid: {p: 100.0, i: 0.1, d: 10.0}